hnn_core/viz.py

```diff
@@ -246,54 +246,60 @@ def plot_dipole(dpl, tmin=None, tmax=None, ax=None, layer='agg', decim=None,
     import matplotlib.pyplot as plt
     from .dipole import Dipole, average_dipoles
 
+    layers = layer if isinstance(layer, list) else [layer]
     if ax is None:
-        _, ax = plt.subplots(1, 1, constrained_layout=True)
+        _, ax = plt.subplots(len(layers), 1, constrained_layout=True)
```

Comment:
I think the y-axis should be shared so that the contributions of different layers can be compared.

![image](https://user-images.githubusercontent.com/15852194/164129260-146e9417-b5ae-417a-b4c3-2d53de8df6a3.png)

You can do `sharey=True`. Maybe also `sharex=True`. Will make the plot more compact looking